toolhub/templates/home.jinja

```diff
@@ -1,5 +1,61 @@
 {% extends "layouts/base.jinja" %}
+{% from "partials/badges.jinja" import taxonomy_list, availability_badge %}
```

Comment:
Let's add a `head_title` block to the header and put something like `{{ _("Home") }}` in it.